mobile/profile.go

```diff
@@ -47,3 +58,117 @@ func (m *Mobile) Avatar() (string, error) {
 
 	return m.node.Avatar(), nil
 }
+
+// SetAvatar calls core SetAvatar
+func (m *Mobile) SetAvatar(path string) error {
```

Comment:
Looks like this returns nothing or an error. How does remote pinning the data happen? I guess I still expected to have to handle the pin request. Or are you only sending that pin request through the cafe request queue now?

Reply:
Ahhhh wow. I completely forgot about the mobile pin step. That's why we have the two methods, prepare and add. Once the queue stuff is in place, this will go away. Thinking about how to proceed in the meantime...

Reply:
We might be best off to just leave the avatar handling as is for now... so you just have to add to the avatar thread via prepare and add.